<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MnemonicDialog</class>
 <widget class="QDialog" name="MnemonicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import mnemonic</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>560</width>
     <height>230</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <widget class="QWidget" name="mnemonictab">
    <attribute name="title">
     <string>Import mnemonic</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>540</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input your mnemonic</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="importMnemonic">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>110</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="mnemonicEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>520</width>
       <height>110</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Separating by space</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="privatekeytab">
    <attribute name="title">
     <string>Import privatekey</string>
    </attribute>
    <widget class="QLineEdit" name="privatekeyEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>520</width>
       <height>110</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="placeholderText">
      <string>example: L2pVnppu4S12FDFTn6sCoyvfCU4QiedHVxNyqJQ63Kty6gGRm1Mz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importPrivatekey">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>110</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>540</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input single privatekey</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Import privakeyfile</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>540</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input privatekey file</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="fileEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>460</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="importWallet">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>110</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fileButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>45</y>
       <width>51</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>520</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Please choose your backupwallet file in your folder.
 The wallet file is not *.dat but is a file returned by command &quot;dumpwallet filename&quot;.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="mnemonicInformation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>560</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>btc to god</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
